client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -148,10 +153,17 @@ class EmailProvidersComparison extends Component {
 	};
 
 	goToEmailForwarding = () => {
-		const { currentRoute, selectedDomainName, selectedSite } = this.props;
+		const { context, currentRoute, selectedDomainName, selectedSite } = this.props;
 
 		recordTracksEvent( 'calypso_email_providers_add_click', { provider: 'email-forwarding' } );
 
+		if ( context === 'inbox-management' ) {
```

Comment:
Already mentioned elsewhere, but reiterating. Let's shorten this to `inbox` and use a constant for it.